templates/install_puppet.bat.erb

```diff
@@ -23,9 +23,12 @@ REM This may fail on agents without pxp-agent, but since this is not
 REM run interactively and the next command sets ERRORLEVEL, it's OK.
 net stop pxp-agent
 
-start /wait msiexec.exe /qn /norestart /i "<%= @_msi_location %>" /l*vx "<%= @_logfile %>" PUPPET_MASTER_SERVER="<%= @_puppet_master %>"
+start /wait msiexec.exe /qn /norestart /i "<%= @_msi_location %>" /l*vx "<%= @_logfile %>" PUPPET_MASTER_SERVER="<%= @_puppet_master %>" <% unless @install_dir.empty? -%>INSTALDIR="<%= @install_dir %>"<% end -%>
```

Comment:
This property is `INSTALLDIR`, not `INSTALDIR`
https://docs.puppet.com/puppet/latest/reference/install_windows.html#installdir